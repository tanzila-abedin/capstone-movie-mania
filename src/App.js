function App() {
  return (
    <div className="App">
      <h1>Movie mania</h1>
    </div>
  );
}

export default App;
