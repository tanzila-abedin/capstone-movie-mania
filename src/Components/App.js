import { BrowserRouter, Routes, Route } from 'react-router-dom';
import MovieList from '../Containers/MovieList';
import Navigate from './navigate';

import SingleMovie from '../Containers/singleMovie';
import Footer from './footer';

function App() {
  return (
    <BrowserRouter>
      <Navigate />
      <Routes>
        <Route exact path="/" data-testid="movieList" element={<MovieList />} />
        <Route path="/singleMovie/:id" element={<SingleMovie />} />
      </Routes>
      <Footer />
    </BrowserRouter>
  );
}

export default App;
