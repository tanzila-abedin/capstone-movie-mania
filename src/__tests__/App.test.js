import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import App from '../Components/App';
import store from '../store';

test('App snapshot with react and redux  ', () => {
  const app = renderer.create(
    <Provider store={store}>
      <App />
    </Provider>,
  ).toJSON();
  expect(app).toMatchSnapshot();
});
