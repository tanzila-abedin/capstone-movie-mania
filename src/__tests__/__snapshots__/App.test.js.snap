// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot with react and redux   1`] = `
Array [
  <nav
    className="navHead navbar navbar-light"
  >
    <div
      className="container"
    >
      <p
        className="navbar-brand mb-0 text-warning nav-text font-weight-normal"
      >
        Movie Mania
      </p>
      <a
        className="nav-text font-weight-normal"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </div>
  </nav>,
  <div
    className="container"
  >
    <div
      className="mt-5 text-center"
    >
      <div
        className="container pb-5 form-bg"
      >
        <h1
          className="display-4 mb-3 text-white"
        >
           
          Search for a popular movie
        </h1>
        <form
          id="searchForm"
        >
          <input
            className="form-control"
            onChange={[Function]}
            placeholder="start typing to filter.."
            type="search"
            value=""
          />
        </form>
      </div>
    </div>
    <div
      className="row"
    />
    <div
      className="row filter-body"
    />
  </div>,
  <nav
    className="navbar navbar-expand-sm navHead navHead  fixed-bottom"
    data-testid="footer"
  >
    <div
      className="container text-center text-light d-flex justify-content-center"
    >
      Built By:
      <span
        className="text-warning font-weight-normal"
      >
        Tanzila Abedin
      </span>
      , Using
       
      <a
        className="text-warning "
        href="https://imdb-api.com/"
        rel="noopener noreferrer"
        target="_blank"
      >
        ImDBI
      </a>
      API
    </div>
  </nav>,
]
`;
